thomas/index.html

```diff
@@ -29,7 +29,27 @@
         var fish = 0;
         function addFish() {
             fish = fish+1;
-            alert(fish);
+           type="text/javascript"> 
+
+ 
+-			// Show them how many fish they have 
+
+-			displayfish(); 
+
+
+
+ 
+-		// Display fish
+
+
+
+ 
+-		function displayFish() { 
+
+
+
+ 
+-			e_fishQuantity.innerHTML = fish; 
```

Comment:
You haven't made an element with id of `e_fishQuantity`. You could add the line:
```
<p>You have <span id="e_fishQuantity>0</span> fish</p>
```
Under your image (`img`) tag, on line 27